server/item/elytra.go

```diff
@@ -0,0 +1,56 @@
+package item
+
+import "github.com/df-mc/dragonfly/server/world"
+
+// Elytra is a pair of rare wings found in end ships that are the only single-item source of flight in Survival mode.
+type Elytra struct{}
+
+// Use handles the using of an elytra to auto-equip it in an armour slot.
+func (Elytra) Use(_ *world.World, _ User, ctx *UseContext) bool {
+	ctx.SwapHeldWithArmour(1)
+	return false
+}
+
+// DurabilityInfo ...
+func (e Elytra) DurabilityInfo() DurabilityInfo {
+	return DurabilityInfo{
+		MaxDurability: 433,
+		BrokenItem:    simpleItem(NewStack(e, 1).WithDurability(1)),
```

Comment:
This wipes all item data, i.e enchantments, custom names

Reply:
Not sure how to properly implement this then... Any ideas @Sandertv? Or should we just bring back the "Persistent" field.

Reply:
Hmmm yeah that's an issue... I guess the easiest solution would be to bring that field back.

Reply:
Updated.